torch/csrc/inductor/aoti_torch/oss_proxy_executor.cpp

```diff
@@ -15,6 +15,86 @@ at::Tensor* tensor_handle_to_tensor_pointer(AtenTensorHandle handle) {
 namespace torch {
 namespace aot_inductor {
 
+c10::ScalarType OSSProxyExecutor::convertSerializedScalarType(int scalarType) {
+  // Numberings match with torch._export.serde.schema.ScalarType enum
+  switch (scalarType) {
+    case 0: // ScalarType.UNKNOWN
+      TORCH_CHECK(false, "Unknown scalar type");
+    case 1: // ScalarType.BYTE
```

Comment:
We need to make sure this kind of enum value mapping is always valid, to avoid any surprising BC breakage. I think this is achieved by defining a Thrift mapping internally (or maybe  even that doesn't provide a strong guard on BC, cc @SherlockNoMad ). Because this kind of enum value should rarely change in practice, we could add some enum value checking at the compile time to make sure the mapping hasn't changed. Or maybe simply adding more tests to test/cpp/aoti_abi_check to prevent surprising enum value changes.